src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechSynthesis.java

```diff
@@ -24,6 +24,7 @@
 import org.myrobotlab.service.interfaces.KeyConsumer;

```

Comment:
Changes to speech synthesis should be done in a different pull request from the service life cycle. please

Reply:
These file only get changed updated, because I'm keeping it in sync with develop.  These are merge conflicts that have been resolved, because changes occurred on this file and were pushed into develop from another pr.